src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const array = Object.values(robot);
+
+  for (let i = 0; i < array.length - 1; i++) {
+    for (let j = i + 1; j < array.length; j++) {
+      if (array[i] === array[j]) {
+        return null;
+      }
+    }
+  }
+
+  const result = Object.fromEntries(Object.entries(robot)
+    .map(([key, value]) => [value, key]));
+
+  return result;
```

Comment:
actually, you can use just 1 loop
you need to create new empty object, and add new inversed values, if you haven't already if, if you already have - return null, that's all)